### PR TITLE
Add a new Handling Touches guide

### PR DESCRIPTION
**Pull request by hramos**
The new Handling Touches guide provides an overall view of how touches can be handled. It is meant to be a higher level discussion of basic touch handling, e.g. "how do I implement a button?". The existing Gesture Responder System guide has been moved to the end of the docs and is still available for reference when building custom gesture handlers.

Reference: #8160 

handlingtouchesguide

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @caabernathy and @grabbou to be potential reviewers.

File: docs/HandlingTouches.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: Handling Touches
+layout: docs
+category: Guides
+permalink: docs/handling-touches.html
+next: animations
+---
+
+Users interact with mobile apps mainly through touch. They can use a combination of gestures, such as tapping on a button, scrolling a list, or zooming on a map.
+
+React Native provides components to handle common gestures, such as taps and swipes, as well as a comprehensive gesture responder system to allow for more advanced gesture recognition.
```
Comment (JoelMarcey) [inline docs/HandlingTouches.md]: nit: link to the "gesture responder system" guide here?

File: docs/HandlingTouches.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- If you need to handle a tap gesture but you don't want any feedback to be displayed, use [**TouchableWithoutFeedback**](/docs/touchablewithoutfeedback.html).
+
+### Long presses
+
+In some cases, you may want to detect when a user presses and holds a view for a set amount of time. These long presses can be handled by passing a function to the `onLongPress` props of any of the touchable components listed above.
+
+## Scrolling lists and swiping views
+
+A common pattern to many mobile apps is the scrollable list of items. Users interact with these using panning or swiping gestures. The [ScrollView](/docs/scrollview.html) component displays a list of items that can be scrolled using these gestures.
```
Comment (JoelMarcey) [inline docs/HandlingTouches.md]: overarching question: Should this link to the actual `ScrollView` API or the `ScrollView` basics tutorial instead?

File: docs/HandlingTouches.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+ScrollViews can scroll vertically or horizontally, and can be configured to allow paging through views by swiping. Swiping horizontally between views can also be implemented on Android using the [ViewPagerAndroid](/docs/viewpagerandroid.html) component.
+
+A [ListView](/docs/listview.html) is a special kind of ScrollView that is best suited for displaying long vertical lists of items. It can also display section headers and footers, similar to `UITableView`s on iOS.
```
Comment (JoelMarcey) [inline docs/HandlingTouches.md]: DItto question above for `ListView`. I think either is probably fine -- so nothing to really change here, but just wondering how we choose.

File: docs/HandlingTouches.md (hunk 2 above)
Comment (hramos) [inline docs/HandlingTouches.md]: It does make sense to link to the Basics tutorial in both cases instead.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/501525576639039/int_phab) to review.

**Pull request closed without merging**